packages/next-image-plus/src/background-image.tsx

```diff
@@ -102,22 +108,26 @@ interface StyleProps {
  *
  * @returns A `<style>` element containing the computed background image CSS rules.
  */
-function Style({ id, bgImageProps }: StyleProps) {
-  // Get the smallest media query as the fallback.
-  const mediaQueries = [];
-  for (const [_key, props] of Object.entries(bgImageProps)) {
-    mediaQueries.push(props.media);
+export function Style({ id, bgImageProps }: StyleProps) {
+  let fallbackMediaQuery = null;
+  // We only need media queries if there's more than 1 image.
+  if (Object.keys(bgImageProps).length > 1) {
+    // Get the smallest media query as the fallback.
+    const mediaQueries = [];
+    for (const [_key, props] of Object.entries(bgImageProps)) {
```

Comment:
Check if media is null and throw error:

Background image with src "${src}" is missing required "media" property. Either use a single background image, or add a media query.